packages/driver/src/cy/commands/sessions/utils.ts

```diff
@@ -97,39 +98,48 @@ const setPostMessageLocalStorage = async (specWindow, originOptions) => {
   })
 }
 
-const getConsoleProps = (sessState: SessionData) => {
-  const sessionDetails = getSessionDetailsForTable(sessState)
-
-  const tables = _.flatMap(sessionDetails, (val, domain) => {
-    const cookiesTable = () => {
-      return {
-        name: `🍪 Cookies - ${domain} (${val.cookies.length})`,
-        data: val.cookies,
-      }
-    }
-
-    const localStorageTable = () => {
-      return {
-        name: `📁 Storage - ${domain} (${_.keys(val.localStorage.value).length})`,
-        data: _.map(val.localStorage.value, (value, key) => {
-          return {
-            key,
-            value,
-          }
-        }),
-      }
+const getConsoleProps = (session: SessionData) => {
+  const sessionDetails = getSessionDetailsByDomain(session)
+
+  const groupsByDomain = _.flatMap(sessionDetails, (val, domain) => {
+    return {
+      name: `${domain} data:`,
+      expand: true,
+      label: false,
+      groups: _.compact([
+        val.cookies && {
+          name: `🍪 Cookies - (${val.cookies.length})`,
+          expand: true,
+          items: val.cookies,
+        },
+        val.localStorage && {
+          name: `📁 Local Storage - (${_.keys(val.localStorage.value).length})`,
+          label: true,
+          expand: true,
+          items: val.localStorage.value,
+        },
+        val.sessionStorage && {
+          name: `📁 Session Storage - (${_.keys(val.sessionStorage.value).length})`,
+          expand: true,
+          label: true,
+          items: val.sessionStorage.value,
+        },
+      ]),
     }
-
-    return [
-      val.cookies && cookiesTable,
-      val.localStorage && localStorageTable,
-    ]
   })
 
-  return {
-    id: sessState.id,
-    table: _.compact(tables),
+  const props = {
+    id: session.id,
+    ...(!groupsByDomain.length && {
+      Warning: '⚠️ There are no cookies, local storage or session storage associated to this session.',
```

Comment:
would this read better?
```suggestion
      Warning: '⚠️ There are no cookies, local storage nor session storage associated with this session.',
```

Reply:
[`4fe9cf0` (#24511)](https://github.com/cypress-io/cypress/pull/24511/commits/4fe9cf0ae1644bea7ea2fc00fa06d3986e63195f)